### PR TITLE
Add help icon to launcher

### LINKED ISSUE #924: Add a link to the elyra documentation to the JupyterLab UI
Currently users cannot access the Elyra documentation (https://elyra.readthedocs.io/en/latest/?badge=latest) without leaving the JupyterLab UI. To find out where the documentation is located one has to open https://github.com/elyra-ai/elyra and locate the `docs:passing` badge. 
To make it easier for users to access the documentation (since there is no embedded help) we should add an easy to find link to the UI that opens the doc in the default browser (whatever might be configured in JupyterLab to handle HTML content).

One highly visible location would be the Elyra category in the Launcher, which already includes links to a couple of Elyra extensions.

image

### PR DESCRIPTION
**Pull request by lresende**
Fixes #924 
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by lresende** [comment]
image

**Comment by lresende** [comment]
@ptitzler are we ok for the icon? Also, are we ok on opening the help page on another browser tab?

**Comment by ptitzler** [comment]
Yes, I like it! Perhaps use "Open documentation" as the label?

**Comment by lresende** [comment]
image

**Review by ptitzler — APPROVED** [review]
LGTM. works fine

File: packages/theme/src/index.ts [hunk 1]
```diff
@@ -114,6 +117,20 @@ const extension: JupyterFrontEndPlugin<ILauncher> = {
       palette.addItem({ command: CommandIDs.create, category: 'Launcher' });
     }
 
+    commands.addCommand(CommandIDs.openHelp, {
+      label: 'Documentation',
+      icon: helpIcon,
+      execute: (args: any) => {
+        window.open('https://elyra.readthedocs.io/en/latest/', '_blank');
```
Comment (ajbozarth) [inline packages/theme/src/index.ts:124]: Do we want this to always open latest? Some sort of versioning or using stable might be better?

**Review by ajbozarth — COMMENTED** [review]
Overall solid PR, one question on the doc link used, and the icon needed some work. I'll be pushing an update to the icon and a new screenshot right after this.

File: packages/theme/src/index.ts (hunk 1 above)
Comment (lresende) [reply packages/theme/src/index.ts:124]: Master -> latest

Comment (ajbozarth) [reply packages/theme/src/index.ts:124]: yea, but then when a user installs Elyra 1.X in a couple months it would open 2.X documentation

**Comment by ajbozarth** [comment]
updated icon to support dark mode
Screen Shot 2020-10-09 at 2 02 10 PM
Screen Shot 2020-10-09 at 2 02 17 PM

File: packages/theme/src/index.ts (hunk 1 above)
Comment (lresende) [reply packages/theme/src/index.ts:124]: Yes, the release process need to handle this

**Comment by lresende** [comment]
Thanks for the support for dark mode @ajbozarth

File: packages/theme/src/index.ts (hunk 1 above)
Comment (lresende) [reply packages/theme/src/index.ts:124]: I have updated the release checklist to take care of this
https://github.com/elyra-ai/elyra/issues/503

**Review by ajbozarth — APPROVED** [review]

**Pull request merged**